tools/release/image-mirror/mirror.go

```diff
@@ -15,28 +15,17 @@ import (
 )
 
 const (
-	ghcr                 = "ghcr.io"
-	gcr                  = "gcr.io"
+	quay				 = "quay.io"
 	defaultSleepDuration = 60 * time.Second
 )
 
 var (
 	httpClient = &http.Client{Timeout: 10 * time.Second}
 )
 
-// GHCRTagsResponse contains the tags' information of the HTTP get response from ghcr.io.
-type GHCRTagsResponse struct {
-	Tags []string `json:"tags"`
-}
-
-// GCRTagsResponse contains the tags' information of the HTTP get response from gcr.io.
-type GCRTagsResponse struct {
-	Tags []string `json:"tags"`
-}
-
-// GHCRToken contains the necessary token information to get an HTTP response from ghcr.io
-type GHCRToken struct {
-	Token string
+// QuayTagsResponse contains the tags' information of the HTTP get response from quay.io.
+type QuayTagsResponse struct {
+	Tags []RepositoryTag `json:"tags"`
```

Comment:
Why doesn't this follow the same structure as the GHCR or GRC response structs?

Reply:
Yeah, the api response is different from others as it is giving the [repo tags](https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/red_hat_quay_api_guide/index#listrepotags) which is leading to `cannot unmarshal object` error